recipes/dlo-hic-tools/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Instead of a `build.sh` that calls `setup.py install`, you can also add
```yaml
  script: python -m pip install --no-deps --ignore-installed .
```
to the `build` section in `meta.yaml` and replace `setuptools` by `pip` in `requirements/host`. This might helps us in the future with automatic maintenance work.